packages/data-api/src/fetchData.js

```diff
@@ -6,63 +6,159 @@ import { utcMoment } from '@tupaia/utils';
 
 import { SqlQuery } from './SqlQuery';
 
-const generateBaseSqlQuery = ({ dataElementCodes, organisationUnitCodes, startDate, endDate }) => {
+const SUPPORTED_AGGREGATIONS = [
```

Comment:
Really not my cleanest code throughout this file 😅 will clean it up soon I promise